metisp/pymetis/src/pymetis/recipes/metis_det_dark.py

```diff
@@ -85,6 +88,18 @@ class GainMapInput(OptionalInputMixin, GainMapInput):
     # but it enables us to introspect the class for the manpage and DRLD.
     ProductMasterDark = MasterDark
 
+    QcDarkMedian = DarkMedian
+    QcDarkMean = DarkMean
+    QcDarkRms = DarkRms
+    QcDarkNBadpix = DarkNBadpix
+    QcDarkNColdpix = DarkNColdpix
+    QcDarkNHotpix = DarkNHotpix
+    QcDarkMedianMean = DarkMedianMean
+    QcDarkMedianMedian = DarkMedianMedian
+    QcDarkMedianRms = DarkMedianRms
+    QcDarkMedianMin = DarkMedianMin
+    QcDarkMedianMax = DarkMedianMax
+
```

Comment:
Is it a thought to perhaps create a `QC` class that collects all these parameters (well more like attributes in Python parlance)?

Perhaps not now, but I've learned that at some point these classes can get some many attribute that it becomes a bit hard to disentangle everything. But maybe we should cross that bridge when we get there.